app/models/widget.rb

```diff
@@ -1,2 +1,6 @@
 class Widget < ApplicationRecord
+  def bad_method
+    unused_var = "This never gets used"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.